login-workflow/example3/src/actions/AuthUIActions.tsx

```diff
@@ -14,7 +15,7 @@ function isRandomFailure(): boolean {
 }
 
 type AuthUIActionsFunction = () => AuthUIActions;
-type AuthUIActionsWithSecurity = (securityHelper: SecurityContextActions) => AuthUIActionsFunction;
+type AuthUIActionsWithSecurity = (appHelper: AppContextType) => AuthUIActionsFunction;
```

Comment:
As we have removed security and using app context instead, we should rename this to something similar to : AuthUIActionsWithApp